roles/kafka_controller/tasks/main.yml

```diff
@@ -100,14 +100,15 @@
 
 - name: Copy Kafka broker's Service to Create kafka Controller's service
   copy:
-    src: "{{systemd_base_dir}}/{{kafka_broker.systemd_file|basename}}"
+    src: "{{ ((binary_base_path, 'lib/systemd/system') | path_join if installation_method=='archive' else systemd_base_dir, kafka_broker.systemd_file|basename) | path_join }}"
```

Comment:
Do you see this as bug in kafka broker task ? Why don't we update the systemd_base_dir in vars/main.yml so that the updated value can be used any other place as well